From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@hveiras__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

tysonn commented:
@dragon119 - Masashi, can you review this contribution please?

dragon119 commented:
#sign-off

Sammak committed:
Merge pull request #2 from MicrosoftDocs/master

Update

Commit:
task 950: Updates for ADN rules engine distinctions, acrolinx edits

ShawnJackson committed:
edit pass: monitoring-overview

Commit:
task 950: Updates for ADN rules engine distinctions, acrolinx edits

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into monitoring-overview

daveba committed:
adding new files for enabling msi on vmss

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into configure-msi-for-azure-vmss

daveba committed:
fixing toc file

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into configure-msi-for-azure-vmss

ShawnJackson committed:
edit pass: monitoring-overview

ShawnJackson committed:
edit pass: monitoring-overview

Commit:
task 950: Add procedures, edits, update screenshots

Commit:
task 950: Edits, add rule engine link, shrink screenshot

Commit:
task 950: Incorporate review comments, add screenshots

ShawnJackson committed:
edit pass: grow-your-business-azure-marketplace

Commit:
task 950: edits

daveba committed:
updating the TOC and consolidating the docs into one docset

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into configure-msi-for-azure-vmss

daveba committed:
corrections based on feedback

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into grow-your-business-azure-marketplace

ShawnJackson committed:
edit pass: grow-your-business-azure-marketplace

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into grow-your-business-azure-marketplace

ShawnJackson committed:
edit pass: grow-your-business-azure-marketplace

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into grow-your-business-azure-marketplace

ShawnJackson committed:
edit pass: grow-your-business-azure-marketplace

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into grow-your-business-azure-marketplace

rolyon committed:
Added rbac warning when transfering subscription

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-rbacwarn

daveba committed:
fixing link

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into configure-msi-for-azure-vmss

ShawnJackson committed:
edit pass: grow-your-business-azure-marketplace

ShawnJackson committed:
edit pass: grow-your-business-azure-marketplace

Sammak committed:
Merge pull request #3 from MicrosoftDocs/master

Updating fork

rolyon committed:
Repositioned RBAC warning, updated wording to across tenants

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-rbacwarn

rolyon committed:
Moved RBAC warning before Specify New Owner screenshot

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-rbacwarn

rolyon committed:
Duplicated RBAC note at top of article

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-rbacwarn

Sammak committed:
Updating documentation for language customization

GitHubber17 committed:
edit pass - Bing Search articles - draft 1

GitHubber17 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vsts-1192661

msmbaldwin committed:
Fixed typos, added to TOC

daveba committed:
making updates based on feedback

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into configure-msi-for-azure-vmss

eringreenlee committed:
added alerts

add alerts AADDS500-AADDS504

GitHubber17 committed:
edit pass: adjustments

eringreenlee committed:
Merge branch 'master' into master

eringreenlee committed:
fixed grammar in troubleshooting alerts

eringreenlee committed:
Merge branch 'master' of https://github.com/eringreenlee/azure-docs-pr

Sammak committed:
Correcting typos

j-martens committed:
Cleaning up the resources TOC section to remove extraneous entries and simplify.

j-martens committed:
acrolinx cleanup

j-martens committed:
fixing capitalization and headings

ajlam committed:
Merge pull request #13 from MicrosoftDocs/master

Update fork

Sammak committed:
Updating doc for blocking issues

Commit:
adding line showing support for 1709

Commit:
updating date

Commit:
formatting fix

JasonWHowell committed:
fixing screenshots

j-martens committed:
Fix for PR

rachel-msft committed:
changes to restore portal articles

Commit:
Initial drafts.

j-martens committed:
AML: Usability updates for Tutorial 1

Commit:
Update data-lake-analytics-overview.md

Commit:
Updates.

anrothMSFT committed:
including new azure key content

JasonWHowell committed:
Updating screenshots

mumian committed:
[HDI] incorporate usability study feedback

MikeDodaro committed:
Fix 3 problems in supported markets

j-martens committed:
AML: formatting fix

mumian committed:
fix a link

v-geberr committed:
importing 1.x for luis

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 0301-migration-guide

MikeDodaro committed:
Change Tom Cruise to Harry Potter

JasonWHowell committed:
Updating screenshots

GitHubber17 committed:
edit pass: adjustments

GitHubber17 committed:
Merge pull request #34164 from Sammak/master

Language customization updates

MikeDodaro committed:
Tom Cruise -> Harry Potter

JasonWHowell committed:
Updating screenshots

PRMerger12 committed:
Merge pull request #34294 from MikeDodaro/typos

Fix 3 problems in supported markets

PRMerger12 committed:
Merge pull request #34287 from anrothMSFT/patch-2

including new azure key content

bandersmsft committed:
cost mgt - updated Activate Azure subscriptions

v-geberr committed:
updates

GitHubber17 committed:
Merge pull request #34276 from rachel-msft/editr

changes to restore portal articles

v-geberr committed:
links

PRMerger15 committed:
Merge pull request #34299 from bandersmsft/master

cost mgt - updated Activate Azure subscriptions

Commit:
Removed article.

ajlam committed:
Add check extension version

mumian committed:
update a screenshot

ShawnJackson committed:
Update grow-your-business-azure-marketplace.md

v-geberr committed:
edits

ajlam committed:
Update tutorial with extension version check

JasonWHowell committed:
Updating screenshots

Commit:
Updated link and acrolinx fixes

daveirwin1 committed:
Updated for 9.4.2

CarlRabeler committed:
Update sql-database-elastic-pool.md

CarlRabeler committed:
Update sql-database-elastic-pool.md

ajlam committed:
Add update for PG

PRMerger10 committed:
Merge pull request #34306 from jeffgilb/jeff31

Updated link and acrolinx fixes

Commit:
automation : delete missing vm change article

PRMerger12 committed:
Merge pull request #34282 from saveenr/patch-20

Update data-lake-analytics-overview.md

eringreenlee committed:
fixed powershell script and picture

for nsg alerts

Commit:
vnet-tutorials

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into virtual-network

GitHubber17 committed:
Merge pull request #34300 from jimdial/remove-vnet-article

Removed article.

PRMerger15 committed:
Merge pull request #34304 from ajlam/master

Add extension version check to MySQL and PG quickstart & tutorials

PRMerger17 committed:
Merge pull request #34307 from CarlRabeler/patch-7

Update sql-database-elastic-pool.md

Commit:
updated

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into virtual-network

PRMerger18 committed:
Merge pull request #34308 from daveirwin1/master

Updated for 9.4.2

vidhoonv committed:
Add samples for Mongoose and spring boot

Linking new sample repos to documentation article.

PRMerger19 committed:
Merge pull request #34021 from rolyon/rolyon-rbacwarn

[AzureAD] RBAC - Warning when transferring a subscription

v-geberr committed:
links

Commit:
Merge pull request #34296 from MikeDodaro/fixAmbig

Change Tom Cruise to Harry Potter

Ja-Dunn committed:
Merge pull request #34036 from ShawnJackson/grow-your-business-azure-marketplace

edit pass: grow-your-business-azure-marketplace

Commit:
Merge pull request #34313 from vidhoonv/patch-2

Add samples for Mongoose and spring boot

mrbullwinkle committed:
Add nuget.org

ktoliver committed:
Merge pull request #34291 from mumian/0223-usability

[HDI] incorporate usability study feedback

bandersmsft committed:
cost mgt - updated understanding cost reports

PRMerger12 committed:
Merge pull request #34305 from v-geberr/0301-dashboard

updates

Commit:
Aligned to contributor guide

PRMerger13 committed:
Merge pull request #34295 from v-geberr/0301-migration-guide

0301 migration guide

Commit:
Fix

ShawnJackson committed:
Merge pull request #34174 from GitHubber17/vsts-1192661

edit pass: Bing Search articles

Commit:
Updated next steps.

ktoliver committed:
Merge pull request #34273 from JasonWHowell/master

Fixing screenshots

Ja-Dunn committed:
Merge pull request #33688 from ShawnJackson/monitoring-overview

edit pass: monitoring-overview

v-geberr committed:
remove publish step

markgalioto committed:
adding System State to Azure Backup components table in Intro article

PRMerger19 committed:
Merge pull request #34319 from mrbullwinkle/mrb_03_01_2018

Add nuget.org

christiankuhtz committed:
move away from VIP language

align with general product language (no VIP). probably more updates required here.

Commit:
Merge pull request #34318 from jimdial/patch-80

Aligned to contributor guide

Commit:
Merge pull request #34324 from markgalioto/ab-3-1-2018

adding System State to Azure Backup components table in Intro article

Lisaco88 committed:
Merge pull request #34244 from j-martens/master

Cleaning up the resources sub-TOC

christiankuhtz committed:
added subscription move limitation

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-950

Lisaco88 committed:
Merge pull request #34310 from georgewallace/autovmfixes

automation : delete missing vm change article

christiankuhtz committed:
tweak

neilpeterson committed:
Revert "Syntax fix for $, adding info on JMESPath queries"

PRMerger10 committed:
Merge pull request #34325 from christiankuhtz/patch-57

added subscription move limitation

Lisaco88 committed:
Merge pull request #34240 from eringreenlee/master

added alerts

PRMerger12 committed:
Merge pull request #34262 from dkkapur/master

adding line showing support for 1709

PRMerger13 committed:
Merge pull request #34326 from neilpeterson/revert-34166-k8s-deploy-cluster

Revert "Syntax fix for $, adding info on JMESPath queries"

Commit:
task 950: standardize titles for optimization-related articles

klaaslanghout committed:
Update storage-files-release-notes.md

PRMerger15 committed:
Merge pull request #34323 from christiankuhtz/patch-56

move away from VIP language

daveba committed:
fixing issues found by pr reviewer

PRMerger16 committed:
Merge pull request #34316 from bandersmsft/master

cost mgt - updated understanding cost reports

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into configure-msi-for-azure-vmss

PRMerger17 committed:
Merge pull request #34215 from msmbaldwin/healthblueprint

Fixed typos, added to TOC

PRMerger18 committed:
Merge pull request #34329 from klaaslanghout/patch-14

to release notes, adding that we block resource sub move

ktoliver committed:
Merge pull request #34311 from jimdial/virtual-network

Virtual network

v-geberr committed:
fix link

PRMerger13 committed:
Merge pull request #34334 from v-geberr/0301-fix-link

fix link

Ja-Dunn committed:
Merge pull request #33588 from dksimpson/dean-950

task 950: Updates for ADN rules engine distinctions, acrolinx edits

PRMerger13 committed:
Merge pull request #34321 from v-geberr/0301-batch-testing

remove publish step

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #34338 from MicrosoftDocs/FromPublicRepo

From public repo

Lisaco88 committed:
Merge pull request #33686 from daveba/configure-msi-for-azure-vmss

Configure msi for azure vmss
